great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -2355,9 +2371,9 @@ def delete_expectation_suite(
     @deprecated_argument(argument_name="ge_cloud_id", version="0.15.45")
     def get_expectation_suite(
         self,
-        expectation_suite_name: Optional[str] = None,
-        include_rendered_content: Optional[bool] = None,
-        ge_cloud_id: Optional[str] = None,
+        expectation_suite_name: str | None = None,
+        include_rendered_content: bool | None = None,
+        ge_cloud_id: str | None = None,
```

Comment:
This is deprecated - only name is support moving forward so I'm not going to touch this signature